cmd/aws-janitor/main.go

```diff
@@ -53,6 +53,8 @@ var (
 	enableVPCEndpointsClean    = flag.Bool("enable-vpc-endpoints-clean", false, "If true, clean vpc endpoints.")
 	skipRoute53ManagementCheck = flag.Bool("skip-route53-management-check", false, "If true, skip managed zone check and managed resource name check.")
 	enableDNSZoneClean         = flag.Bool("enable-dns-zone-clean", false, "If true, clean DNS zones.")
+	enableS3BucketClean        = flag.Bool("enable-s3-bucket-clean", false, "If true, clean S3 bucket.")
+	enableStateBucketCheck     = flag.Bool("enable-state-bucket-check", false, "If true, only buckets tagged with exclude-tags can be used to store janitor state data.")
```

Comment:
We probably can drop this flag and just always check tagging if s3 bucket clean is requested. Hard to imagine case when s3 clean is on but we don't want to check.